tools/accuracy_checker/accuracy_checker/annotation_converters/_nlp_common.py

```diff
@@ -64,6 +68,44 @@ def basic_tokenizer(self, text):
         output_tokens = output_tokens.split() if output_tokens else []
         return output_tokens
 
+    def _tokenize_chinese_chars(self, text):
+        """Adds whitespace around any CJK character."""
+        output = []
+        for char in text:
+            cp = ord(char)
+            if self._is_chinese_char(cp):
+                output.append(" ")
+                output.append(char)
+                output.append(" ")
+            else:
+                output.append(char)
+        return "".join(output)
+
+    def _is_chinese_char(cp):
```

Comment:
It will not work, you need to make this method static
```suggestion
    @staticmethod
    def _is_chinese_char(cp):
```